whois-api/src/test/java/net/ripe/db/whois/api/rdap/WhoisRdapServiceTestIntegration.java

```diff
@@ -2283,6 +2291,78 @@ public void lookup_org_entity() {
         assertTnCNotice(notices.get(2), "https://rdap.db.ripe.net/entity/ORG-ONE-TEST");
     }
 
+    @Test
+    public void lookup_redactions() {
+        databaseHelper.addObject("" +
+                "mntner:        ANOTHER-MNT\n" +
+                "descr:         Owner Maintainer\n" +
+                "admin-c:       TP1-TEST\n" +
+                "upd-to:        noreply@ripe.net\n" +
+                "auth:          MD5-PW $1$d9fKeTr2$Si7YudNf4rUGmR71n/cqk/ #test\n" +
+                "mnt-by:        OWNER-MNT\n" +
+                "mbrs-by-ref:   OWNER-MNT\n" +
+                "notify:        test@ripe.net\n" +
+                "created:         2011-07-28T00:35:42Z\n" +
+                "last-modified:   2019-02-28T10:14:46Z\n" +
+                "source:        TEST");
+
+        databaseHelper.addObject("" +
+                "organisation:  ORG-ONE-TEST\n" +
+                "org-name:      Organisation One\n" +
+                "org-type:      LIR\n" +
+                "descr:         Test organisation\n" +
+                "address:       One Org Street\n" +
+                "e-mail:        test@ripe.net\n" +
+                "language:      EN\n" +
+                "admin-c:       TP2-TEST\n" + //has email and notify
+                "tech-c:        TP1-TEST\n" +
+                "tech-c:        TP2-TEST\n" + //has email and notify
+                "mnt-ref:       OWNER-MNT\n" +
+                "mnt-ref:       OWNER-MNT\n" +
```

Comment:
multiple mnt-ref with same value ? Can you change the value ?

Reply:
I want to check that there are not two redactions created for same attribute + same value. 
If you think that it is pointless I will

Reply:
I think we need a test with multiple values for same attribute. Rpsl parsing takes care of same value and same attribute

Reply:
Done